cmd/cniplugins/passt-binding/passt-binding-ds.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
```

Comment:
Please consider splitting the introduction of the DS manifest and changes to build system to dedicated commits.